unsorted/mobaxterm

```diff
@@ -1 +0,0 @@
-uncheck randr
```

Comment:
Might want to make sure this is notated down somewhere - I know its in the intro tutorial, but if someone hasn't done the intro tutorial, they will face this problem when using our codebase.

Reply:
This will also be noted in the new README in the main repo.